tools/sonarcloud-analysis

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+TARGET=sonar-scanner-cli-4.7.0.2747-linux.zip
```

Comment:
how large is this file? add it to git-lfs?

Reply:
It's downloaded with curl in few seconds, should be fine for the CI run